node/_tools/suites/parallel/test-path.js

```diff
@@ -0,0 +1,81 @@
+// deno-fmt-ignore-file
+// deno-lint-ignore-file
+
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+// Taken from Node 16.13.0
+// This file is automatically generated by "node/_tools/setup.ts". Do not modify this file manually
+
+// Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+
+// Test thrown TypeErrors
+const typeErrorTests = [true, false, 7, null, {}, undefined, [], NaN];
+
+function fail(fn) {
+  const args = Array.from(arguments).slice(1);
+
+  assert.throws(() => {
+    fn.apply(null, args);
+  }, { code: 'ERR_INVALID_ARG_TYPE', name: 'TypeError' });
+}
+
+typeErrorTests.forEach((test) => {
+  [path.posix, path.win32].forEach((namespace) => {
+    fail(namespace.join, test);
+    fail(namespace.resolve, test);
+    fail(namespace.normalize, test);
+    fail(namespace.isAbsolute, test);
+    fail(namespace.relative, test, 'foo');
+    fail(namespace.relative, 'foo', test);
+    fail(namespace.parse, test);
+    fail(namespace.dirname, test);
+    fail(namespace.basename, test);
+    fail(namespace.extname, test);
+
+    // Undefined is a valid value as the second argument to basename
+    if (test !== undefined) {
+      fail(namespace.basename, 'foo', test);
+    }
+  });
+});
+
+// path.sep tests
+// windows
+assert.strictEqual(path.win32.sep, '\\');
+// posix
+assert.strictEqual(path.posix.sep, '/');
+
+// path.delimiter tests
+// windows
+assert.strictEqual(path.win32.delimiter, ';');
+// posix
+assert.strictEqual(path.posix.delimiter, ':');
+
+// TODO(wafuwafu13): Enable this
+// if (common.isWindows)
+//   assert.strictEqual(path, path.win32);
+// else
+//   assert.strictEqual(path, path.posix);
```

Comment:
debug by `console.log(path)`(my machine is posix)

deno_std
```zsh
~/Desktop/deno_std 
$ deno run -A --unstable node/_tools/require.ts /Users/wafuwafu13/desktop/deno_std/node/_tools/suites/parallel/test-path.js
{
  SEP: "/",
  SEP_PATTERN: /\/+/,
  basename: [Function: basename],
  common: [Function: common],
  delimiter: ":",
  dirname: [Function: dirname],
  extname: [Function: extname],
  format: [Function: format],
  fromFileUrl: [Function: fromFileUrl],
  globToRegExp: [Function: globToRegExp],
  isAbsolute: [Function: isAbsolute],
  isGlob: [Function: isGlob],
  join: [Function: join],
  joinGlobs: [Function: joinGlobs],
  normalize: [Function: normalize],
  normalizeGlob: [Function: normalizeGlob],
  parse: [Function: parse],
  posix: Module {
    basename: [Function: basename],
    delimiter: ":",
    dirname: [Function: dirname],
    extname: [Function: extname],
    format: [Function: format],
    fromFileUrl: [Function: fromFileUrl],
    isAbsolute: [Function: isAbsolute],
    join: [Function: join],
    normalize: [Function: normalize],
    parse: [Function: parse],
    relative: [Function: relative],
    resolve: [Function: resolve],
    sep: "/",
    toFileUrl: [Function: toFileUrl],
    toNamespacedPath: [Function: toNamespacedPath]
  },
  relative: [Function: relative],
  resolve: [Function: resolve],
  sep: "/",
  toFileUrl: [Function: toFileUrl],
  toNamespacedPath: [Function: toNamespacedPath],
  win32: Module {
    basename: [Function: basename],
    delimiter: ";",
    dirname: [Function: dirname],
    extname: [Function: extname],
    format: [Function: format],
    fromFileUrl: [Function: fromFileUrl],
    isAbsolute: [Function: isAbsolute],
    join: [Function: join],
    normalize: [Function: normalize],
    parse: [Function: parse],
    relative: [Function: relative],
    resolve: [Function: resolve],
    sep: "\\",
    toFileUrl: [Function: toFileUrl],
    toNamespacedPath: [Function: toNamespacedPath]
  }
}
```

nodejs/node/test/parallel/test-path.js
```zsh
~/Desktop/node 
$ ./node ./test/parallel/test-path.js
<ref *2> {
  resolve: [Function: resolve],
  normalize: [Function: normalize],
  isAbsolute: [Function: isAbsolute],
  join: [Function: join],
  relative: [Function: relative],
  toNamespacedPath: [Function: toNamespacedPath],
  dirname: [Function: dirname],
  basename: [Function: basename],
  extname: [Function: extname],
  format: [Function: bound _format],
  parse: [Function: parse],
  sep: '/',
  delimiter: ':',
  win32: <ref *1> {
    resolve: [Function: resolve],
    normalize: [Function: normalize],
    isAbsolute: [Function: isAbsolute],
    join: [Function: join],
    relative: [Function: relative],
    toNamespacedPath: [Function: toNamespacedPath],
    dirname: [Function: dirname],
    basename: [Function: basename],
    extname: [Function: extname],
    format: [Function: bound _format],
    parse: [Function: parse],
    sep: '\\',
    delimiter: ';',
    win32: [Circular *1],
    posix: [Circular *2],
    _makeLong: [Function: toNamespacedPath]
  },
  posix: [Circular *2],
  _makeLong: [Function: toNamespacedPath]
}
```